evennia/contrib/base_systems/ingame_reports/README.md

```diff
@@ -0,0 +1,123 @@
+# In-Game Reporting System
+
+This contrib provides an in-game reports system, handling bug reports, player reports, and idea submissions by default. It also supports adding your own types of reports, or removing any of the default report types.
+
+Each type of report has its own command for submitting new reports, and an admin command is also provided for managing the reports through a menu.
+
+## Installation
+
+To install the reports contrib, just add the provided cmdset to your default AccountCmdSet:
+
+```python
+# in commands/default_cmdset.py
+
+from evennia.contrib.base_systems.ingame_reports import ReportsCmdSet
+
+class AccountCmdSet(default_cmds.AccountCmdSet):
+    # ...
+
+    def at_cmdset_creation(self):
+        # ...
+        self.add(ReportsCmdSet)
+```
+
+The contrib also has two optional settings: `INGAME_REPORT_TYPES` and `INGAME_REPORT_STATUS_TAGS`.
+
+The `INGAME_REPORT_TYPES` setting is covered in detail in the section "Adding new types of reports".
+
+The `INGAME_REPORT_STATUS_TAGS` setting is covered in the section "Managing reports".
+
+## Usage
+
+By default, the following report types are available:
+
+* Bugs: Report bugs encountered during gameplay.
+* Ideas: Submit suggestions for game improvement.
+* Players: Report inappropriate player behavior.
+
+Players can submit new reports through the command for each report type, and staff are given access to a report-management command and menu.
+
+### Submitting reports
+
+Players can submit reports using the following commands:
+
+* `bug` - Files a bug report. An optional target can be included, making it easier for devs/builders to track down issues.
+* `report` - Reports a player for inappropriate or rule-breaking behavior. *Requires* a target to be provided - it searches among accounts by default.
+* `idea` - Submits a general suggestion, with no target. It also has an alias of `ideas` which allows you to view all of your submitted ideas.
+
+### Managing reports
+
+The `manage reports` command allows staff to review and manage the various types of reports. It dynamically aliases itself to include whatever types of reports you've configured for your game - by default, this means it makes `manage bugs`, `manage players`, and `manage ideas` available along with the default `manage reports`.
```

Comment:
In case it changes your thought here: it isn't just `manage`, it's literally the string `manage reports`. The report type isn't an argument but a portion of dynamically-added command aliases.

Reply:
Ah, I didn't notice that. That makes it less likely there'll be a clash, indeed. Very well.